examples/llm/txt2kg_rag.py

```diff
@@ -212,6 +212,36 @@ def _process_and_chunk_text(text, chunk_size, doc_parsing_mode):
 
 def get_data(args):
     # need a JSON dict of Questions and answers, see below for how its used
+    if not (os.path.exists(os.path.join(args.dataset, "train.json")) or os.path.join(args.dataset, "corpus")):
```

Comment:
can you make `args.dataset` default to techqa and run this DL pass only if `args.dataset==techqa`?

Reply:
also need to move the files to the dataset dir